CONTRIBUTING.md

```diff
@@ -1,270 +1,59 @@
-# Contributing to Hack Club
+# How To Contribute
 
-Contributions to Hack Club are very welcome, and strongly encouraged (even to
-this document)!
+This is a guide explaining the contribution workflow we use at Hack Club!
 
-## Doing it
+- [Find something you want to change](#find-something-to-change)
```

Comment:
I think making this an ordered list as opposed to an unordered list makes more sense because these steps are in order.